pandas/io/json.py

```diff
@@ -792,7 +792,10 @@ def _recursive_extract(data, path, seen_meta, level=0):
                     if level + 1 > len(val):
                         meta_val = seen_meta[key]
                     else:
-                        meta_val = _pull_field(obj, val[level:])
```

Comment:
I think this should be a keyword, call it ``errors='raise'|'ignore'``. You are defining ``ignore``. Please leave the default as ``raise`` (which is the current behavior).